source/mainnet/net/resources/glossary.rst

```diff
@@ -727,7 +755,7 @@ WebAssembly (Wasm) defines a portable binary-code format and a corresponding tex
 Winning probability
 ===================
 
-The winning probability is the probability that a baker wins in a given slot.
+The winning probability is the probability that a baker wins in a given round.
```

Comment:
This is much simpler in the new consensus: The probability is directly \alpha, which equals the baker's relative stake. In particular, there is no difficulty parameter anymore.